### DISCUSSION TITLE
core.min.css and core.min.js net::ERR_ABORTED 404 (Not Found)

### DISCUSSION THREAD

**Discussion by echoskydev** [rappasoft/laravel-livewire-tables#1586] [Q&A]
GET https://myweb.com/rappasoft/laravel-livewire-tables/core.min.css net::ERR_ABORTED 404 (Not Found)
GET https://myweb.com/rappasoft/laravel-livewire-tables/core.min.js net::ERR_ABORTED 404 (Not Found)

On my MacBook, it works, but it doesn't work on the Linux Ubuntu version 22 live server. 
On my Linux server, I changed the public path to the external Laravel directory. 
I'm unsure if that's the source of the problem.

On Linux, I moved 'rootpath/laravel/public' to 'rootpath/...'.

Please suggest a solution if you think this approach will work.

**Comment by lrljoe** [comment]
When you say you "moved 'rootpath/laravel/public' to 'rootpath/...'.", what do you mean exactly?

**Reply by echoskydev** [reply]
I mean there's been a change. looks like this 
https://github.com/hannanstd/change-laravel-public
